### DISCUSSION TITLE
Kramdown Gem::LoadError

### DISCUSSION THREAD

**Discussion by rbiddis1** [daattali/beautiful-jekyll#1392] [Q&A] [answered]
Hello,

I'm having trouble with load times on my site. Even on the local serve. It's taking as long as 30+ seconds to load everything. When I run jekyll build --profile, I get "You have already activated kramdown 2.4.0, but your Gemfile requires kramdown 2.3.2. Prepending `bundle exec` to your command may solve this. (Gem::LoadError)"

I use bendle exec when I serve and that doesn't help. I tried adding kramdown 2.3.2 to my gemfile but it doesn't help. 

Does anyone have any suggestions? Thanks so much for you help.

**Comment by daattali** [comment, marked as answer]
I can't help with this, but I want to suggest posting on a more generic jekyll forum so that your question will get seen by a lot more people. It doesn't seem too specific to this theme so someone should be able to help you!

**Reply by rbiddis1** [reply]
Thanks so much for the reply. I'll check with another forum.
